install_config/topics/escaping_special_characters.adoc

```diff
@@ -0,0 +1,17 @@
+////
```

Comment:
@bfallonf, @vikram-redhat  what do you think of reusing a note like this?  Should we? Should we make a separate folder for notes?

Reply:
@kalexand-rh isn't that just another module?